src/links/Retry.ts

```diff
@@ -0,0 +1,64 @@
+import * as Observable from 'zen-observable';
+
+import Link, { OperationRequest } from '../';
+
+// linear interval
+export const defaultInterval = (delay, count) => delay;
+
+// example using a class instead of a closure
+export default class Retry extends Link {
+  max: number = 10;
+  delay: number = 300;
+  count: number = 0;
+  interval: (delay: number, count: number) => number;
```

Comment:
Awesome interface for the retryLink. Let's use this in our standard Link library